model/src/application/interaction/mod.rs

```diff
@@ -74,6 +86,33 @@ impl Interaction {
             Self::ModalSubmit(modal) => modal.id,
         }
     }
+
+    /// Returns `true` if the interaction was run in a DM.
```

Comment:
"Whether" is a concise method of saying "Returns `true` if ...", which we use frequently:
```suggestion
    /// Whether the interaction was invoked in a DM.
```

Reply:
It's more concise, but "Returns `true` if..." is what a lot of other `is_` methods start with (I was gonna say every one in std, but `str::is_ascii` contains no `true`/`false` mention on its docs).